OpenLIFULib/OpenLIFULib/user_account_mode_util.py

```diff
@@ -1,7 +1,16 @@
+from typing import Optional, TYPE_CHECKING
+
 from OpenLIFULib.util import (
         get_openlifu_login_parameter_node,
         )
 
+if TYPE_CHECKING:
+    import openlifu.db
+
+def get_current_user() -> "Optional[openlifu.db.User]":
+    """Get the active openlifu user"""
+    return get_openlifu_login_parameter_node().active_user.user
```

Comment:
This currently errors out if there is no `active_user` (i.e. if user account mode is off). It would be more useful for it to return `None` or something informative like that, and the docstring can explain under exactly which circumstances `None` would be returned and under what circumstances a default anonymous User gets returned.

(this function is sort of like a "front end" of the login module internals for other parts of the application to refer to in the future, that's why the docstring is more important)